app/sagas/sync.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import type { State } from 'types/store.types';
+
```

Comment:
I think we should rename to alerts. What do you think about starting to use a new notation?

Reply:
👍